{"stdout":"","stderr":"npm error code ERESOLVE\nnpm error ERESOLVE unable to resolve dependency tree\nnpm error\nnpm error While resolving: yc_directory@0.1.0\nnpm error Found: next@15.0.0-canary.183\nnpm error node_modules/next\nnpm error   next@\"15.0.0-canary.183\" from the root project\nnpm error\nnpm error Could not resolve dependency:\nnpm error peer next@\"^13.2.0 || ^14.0 || ^15.0.0-rc.0\" from @sentry/nextjs@8.54.0\nnpm error node_modules/@sentry/nextjs\nnpm error   @sentry/nextjs@\"^8\" from the root project\nnpm error\nnpm error Fix the upstream dependency conflict, or retry\nnpm error this command with --force or --legacy-peer-deps\nnpm error to accept an incorrect (and potentially broken) dependency resolution.\nnpm error\nnpm error\nnpm error For a full report see:\nnpm error /Users/kotichai/.npm/_logs/2025-02-05T01_09_34_612Z-eresolve-report.txt\nnpm error A complete log of this run can be found in: /Users/kotichai/.npm/_logs/2025-02-05T01_09_34_612Z-debug-0.log\n"}